playbook/app/pb_kits/playbook/data/samples.yml

```diff
@@ -4,4 +4,4 @@
   - registration
   - filter_table
   - collection_detail
-  
```

Comment:
Pages look great!  Empty space here might throw off future yaml operations (subticks are arranged in an array, not sure what happens when there's a blank space).

Reply:
I dont see it in my code, I think since its red it was removed

Reply:
or are you saying i should add it back

Reply:
Oh woops, good call.  Funny that it registered on git as removing a blank and then adding your entry instead of just adding your entry.  Should be all set!